### PR TITLE
Add an example with CDK

### PR DESCRIPTION
**Pull request by adriencaccia**
This pull request if the first part of #345.

### CODE REVIEW

File: examples/swarmion-with-cdk/backend/core/resources/stack.ts [hunk 1]
```diff
@@ -13,6 +14,18 @@ export class CoreStack extends Stack {
...
+
+    const coreApi = new RestApi(this, 'CoreApi', {
+      // the stage of the API is the same as the stage of the stack
+      description: `Core API - ${stage}`,
+      deployOptions: {
+        stageName: stage,
+      },
+    });
+
+    const healthResource = coreApi.root.addResource('health');
```
Comment (guillaumeduboc) [inline examples/swarmion-with-cdk/backend/core/resources/stack.ts:27]: should this ressource be added here or inside the Health function ?

Comment (guillaumeduboc) [reply examples/swarmion-with-cdk/backend/core/resources/stack.ts:27]: we could use [`resourceForPath`](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-apigateway.IResource.html#resourcewbrforwbrpathpath) method

File: examples/swarmion-with-cdk/backend/core/package.json [hunk 2]
```diff
@@ -36,6 +38,7 @@
     "dotenv": "^16.0.3",
     "esbuild": "^0.15.14",
     "eslint": "^8.27.0",
+    "json-schema-to-ts": "^2.6.0",
```
Comment (guillaumeduboc) [inline examples/swarmion-with-cdk/backend/core/package.json:41]: you shouldn't need this if you bump the `@swarmion/serverless-contracts` package

**Comment by fargito** [comment]
Closing this as we are rather going to use the CDK generator from #478

**Pull request closed without merging**